packages/amplify-category-auth/resources/cloudformation-templates/auth-template.yml.ejs

```diff
@@ -158,7 +158,7 @@ Resources:
           RequireNumbers: <%=  props.passwordPolicyCharacters.includes('Requires Numbers') %>
           RequireSymbols: <%= props.passwordPolicyCharacters.includes('Requires Symbols') %>
           RequireUppercase: <%= props.passwordPolicyCharacters.includes('Requires Uppercase') %>
-      <% if (props.usernameAttributes && props.usernameAttributes !== 'username') { %>
+      <% if (props.usernameAttributes && !props.usernameAttributes.includes('username')) { %>
```

Comment:
Because this should be an array sinces it's something that can have multiple values: usernameAttribute**s** and simplify the code to avoid to have to parse the string to figure out if it's an array or not